upstream/solr6/local/92-init-solr.sh

```diff
@@ -148,10 +149,15 @@ function makeConfigs {
                 setOption 'alfresco.corePoolSize' "${ALFRESCO_CORE_POOL_SIZE:-8}" "$CONFIG_FILE_CORE"
                 setOption 'alfresco.doPermissionChecks' "${ALFRESCO_DO_PERMISSION_CHECKS:-true}" "$CONFIG_FILE_CORE"
                 setOption 'solr.suggester.enabled' "$ALFRESCO_SOLR_SUGGESTER_ENABLED" "$CONFIG_FILE_CORE"
-                # SOLR_BACKUP_DIR only useful for ASS v2.* and up
-                if [ $SOLR_VERSION_MAJOR != 1 ]
+                # SOLR_BACKUP_DIR only useful for ASS v2.0.2* and up
+                if [ $SOLR_VERSION_MAJOR = 2 -a $SOLR_VERSION_REV != 0 -a $SOLR_VERSION_REV != 1 ]
                 then
-                    setOption 'solr.backup.dir' "$SOLR_BACKUP_DIR" "$CONFIG_FILE_CORE"
+                    if [ $coreAlfrescoName != $coreName ]
+		    then
+			setOption 'solr.backup.dir' "$SOLR_BACKUP_DIR/$coreName/$coreAlfrescoName" "$CONFIG_FILE_CORE"
```

Comment:
Is this so when you have multiple shards in one Solr server, they all write to a different backup directory instead of overwriting eachother?

Reply:
Yes also needed for the typical alfresco and archive cores